ext/doc/Cassandra/Numeric.php

```diff
@@ -27,7 +27,7 @@
  * @see Float
  * @see Varint
  */
-interface Numeric
+interface Numeric implements Value
```

Comment:
should be `extends` here, since `Numeric` is also an interface, then disregard previous comments